templates/payment/form-installment.php

```diff
@@ -10,15 +10,23 @@
 							<span class="title"><?php echo $backend->provider_name; ?></span><br/>
 							<select id="<?php echo $backend->_id; ?>_installment_terms" name="<?php echo $backend->_id; ?>_installment_terms" class="installment-term-select-box">
 								<option>Select term</option>
-								<?php foreach ( $backend->allowed_installment_terms as $term ) : ?>
-									<option value="<?php echo $term; ?>"><?php echo $term; ?> <?php echo __('months'); ?></option>
+								<?php foreach ( $backend->allowed_installment_terms as $installment_term ) : ?>
+									<option value="<?php echo $installment_term['term']; ?>"><?php echo $installment_term['term']; ?> <?php echo __('months'); ?> ( <?php echo wc_price( $installment_term['monthly_amount'] ); ?> / <?php echo __('months'); ?> )</option>
 								<?php endforeach; ?>
 							</select>
+							<?php if ( ! $viewData['is_zero_interest'] ): ?>
+								<br/><span class="omise-installment-interest-rate">( <?php echo __( 'interest', 'omise' ); ?> <?php echo $backend->interest; ?>% )</span>
```

Comment:
you never know how it will be exactly translated to other languages.
which word will be where depends on the language.

generating strings to translate should be made using mechanisms from wordpress
https://codex.wordpress.org/I18n_for_WordPress_Developers

example:
```
<p>
<?php
$url = 'http://example.com';
$link = sprintf( wp_kses( __( 'Check out this link to my <a href="%s">website</a> made with WordPress.', 'my-text-domain' ), array(  'a' => array( 'href' => array() ) ) ), esc_url( $url ) );
echo $link;
?>
</p>
```

Reply:
@jacstn Well spotted. Eagle eyes 👀

Reply:
@jacstn  👍 nice catch, thanks.
I'm changing it to
```php
echo sprintf(  __( '( interest %g%% )', 'omise' ), $backend->interest );
```